definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1370,5 +1370,12 @@
     "CaseEventID": "FR_TransferredToLocalCourt",
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "R"
+  },
```

Comment:
this should be behind a feature flag, right?

Reply:
Moved it behind feature flag as suggested